### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jamoralp**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/soss/182/)

File: README.md [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
  * **Finished** `soss-rosidl` -- utility for converting rosidl (ROS2) message/service specifications (.msg/.srv files) into middleware translation libraries
  * **Finished** `soss-genmsg` -- utility for converting genmsg (ROS1) message/service specifications (.msg/.srv files) into middleware translation libraries
- * **Finished** `soss-ros1` -- [ROS1 extension for soss](https://github.com/osrf/soss-ros1)
- * **Finished** `soss-ros2` -- ROS2 extension for soss
- * **Finished** `soss-websocket` -- websocket extension for soss
- * *In progress* - `soss-rest server` -- REST API server extension for soss
- * **Finished** `soss-mock` -- a mock middleware used for testing extensions of soss
- * **Finished** `soss-fiware` -- [FIWARE extension for soss](https://github.com/eProsima/SOSS-FIWARE.git) (from eProsima)
- * **Finished** `soss-dds` -- [DDS extension for soss](https://github.com/eProsima/SOSS-DDS.git) (from eProsima)
+ * **Finished** `soss-ros1` -- [ROS1 extension for SOSS](https://github.com/eprosima/soss-ros1)
```
Comment (lauramg15) [inline README.md]:
```suggestion
 * **Finished** `soss-ros1` -- [ROS1 extension for SOSS](https://github.com/eprosima/soss-ros1.git)
```

File: README.md [hunk 2]
```diff
@@ -4,23 +4,23 @@
...
+ * **Finished** `soss-ros2` -- ROS2 extension for SOSS
+ * **Finished** `soss-websocket` -- websocket extension for SOSS
```
Comment (lauramg15) [inline README.md]:
```suggestion
 * **Finished** `soss-websocket` -- WebSocket extension for SOSS
```

**Review by lauramg15 — CHANGES_REQUESTED** [review]

**Review by lauramg15 — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/soss/183/)

**Pull request merged**